File: DTOs/Sources/ConfigServiceDTOs/WebhookPayloadDTO.swift
import Foundation

public struct WebhookPayloadDTO: Codable {
    public var event: String
    public var settings_id: UUID
    public var new_value: SettingValueDTO?
    public var old_value: SettingValueDTO?
    
    
    public init(event: String, settings_id: UUID, new_value: SettingValueDTO? = nil, old_value: SettingValueDTO? = nil) {
        self.event = event
        self.settings_id = settings_id
        self.new_value = new_value
        self.old_value = old_value
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/ConfigServiceDTOs/WebhookPayloadDTO.kt
package net.ipv64.kivop.dtos.ConfigServiceDTOs

import java.util.UUID

data class WebhookPayloadDTO(
    var event: String,
    var settings_id: UUID,
    var new_value: SettingValueDTO?,
    var old_value: SettingValueDTO?,
)
